.github/workflows/docker.yml

```diff
@@ -3,7 +3,6 @@ name: ovn-docker-images
 on:
   push:
   pull_request:
-    branches: [ master ]
     paths: [ "dist/images/Dockerfile*"]
```

Comment:
any reason why you are dropping the `branches`?

Reply:
Nope :man_shrugging: I think I thought it would do something about the barrage of emails I get from my own fork every time I push a commit :joy: